src/functions/enclose.js

```diff
@@ -46,15 +46,24 @@ const htmlBuilder = (group, options) => {
 
         // Add vertical padding
         let vertPad = 0;
+        let ruleThickness = 0;
         // ref: LaTeX source2e: \fboxsep = 3pt;  \fboxrule = .4pt
         // ref: cancel package: \advance\totalheight2\p@ % "+2"
         if (/box/.test(label)) {
-            vertPad = label === "colorbox" ? 0.3 : 0.34;
+            ruleThickness = Math.max(
+                0.04,                       // \fboxrule = .4pt from LaTeX2e
```

Comment:
This is 0.04 b/c `ptPerEm` is `10`.  Could you add a comment so that others know what's going on with the pt vs em conversions that are happening here?

Reply:
The comment is now added in `fontMetrics.js`